src/js/edu-benefits/1990e/config/form.js

```diff
@@ -6,8 +6,9 @@ import {
 import fullSchema1990e from 'vets-json-schema/dist/transfer-benefits-schema.json';
 
 import * as currentOrPastDate from '../../../common/schemaform/definitions/currentOrPastDate';
-import * as fullName from '../../../common/schemaform/definitions/fullName';
-import * as ssn from '../../../common/schemaform/definitions/ssn';
+import * as fullNameCommon from '../../../common/schemaform/definitions/fullName';
```

Comment:
Hm...I know we talked about this in the schemaform sync yesterday (pulling the schema definition directly from the backend, like you do below for fullName, and here separately pulling out the front-end shared uiSchema definition). If the definitions in schemaform/ are only going to export uiSchema, then we could do something like
```
import uiSchema as fullNameUiSchema from '../../../common/schemaform/definitions/fullName';
```
and then below refer to `fullNameUiSchema` instead of `fullNameCommon.uiSchema`

It's a little cumbersome, but maybe more clear. @jbalboni, wondering what you think